utils/api_info_v2.py

```diff
@@ -61,8 +63,20 @@
     # Go!
     r = requests.get(api_info_url, auth=auth)
 
-    #print(r.text)
-
-    # For printing unicode instead of unicode escape sequences
-    jr = json.loads(r.text)
-    pprint(jr)
+    if args.raw:
+        print(r.text)
+    else:
+        try:
+            rj = r.json()
+        except ValueError:
+            print('Bad response:')
+            print(r.text)
+            exit(1)
+
+        errors = rj.get('errors')
+
+        if errors:
+            print('Info request failed:',  errors)
+            exit(1)
+        else:
+            print("Torrent #{} '{}' uploaded by '{}' ({} bytes) {} / {}\n{}".format(rj['id'], rj['name'], rj['submitter'], rj['filesize'], rj['main_category'], rj['sub_category'], rj['magnet']))
```

Comment:
Well now, would you look at that... I didn't know that was possible.
You can refrence the object's elements from the format replacement fields instead of adding each element to the format function.
https://docs.python.org/2/library/string.html#format-string-syntax
```python
>>> t = {}
>>> t = {'one': 1, 'two': 2}
>>> '{0[one]}'.format(t)
'1'
>>>
```